song-server/src/test/java/bio/overture/song/server/service/MockedSubmitTest.java

```diff
@@ -101,7 +101,7 @@ public void submit_StudyDne_StudyIdDoesNotExist() {
     // Verify
     assertSongError(
         () -> submitService.submit("anyStudy", "anyAnalysisId", false), STUDY_ID_DOES_NOT_EXIST);
-    verify(validationService, never()).validate(isA(JsonNode.class));
+    verify(validationService, never()).validate(isA(JsonNode.class),"anyStudy");
```

Comment:
what's the reason we're passing an study to the method `validate`?

Reply:
As we need to populate `studyId` value for external validation, hence we have added this as a parameter. If required "`anyStudy`" to "`anyStudyId`"